apps/application-status/.env.example

```diff
@@ -1 +1,2 @@
-ZENHUB_APIKEY
\ No newline at end of file
+ZENHUB_APIKEY
+GITHUB_TOKEN
```

Comment:
where are you pulling GITHUB_TOKEN from? If you want this variable to be in gcbrun, you will need to add it here: https://github.com/bcgov/sbc-producthub/blob/069ac47b532e8cc61505f57e395ad06b3ec051ad/apps/application-status/devops/cloudbuild-pr.yaml#L10
If you also want it to be loaded in the builds from 1password, you will need to add it here https://github.com/bcgov/sbc-producthub/blob/main/apps/application-status/vaults.env probably something like GITHUB_TOKEN ="op://API/$APP_ENV/github/GITHUB_TOKEN"

Reply:
I sent you direct message about the key value, as I don't know which value you are using